securing_apps/topics/oidc/java/fuse7/classic-war.adoc

```diff
@@ -83,8 +84,31 @@ To enable the functionality, add this section to your `/WEB_INF/web.xml` file:
 </context-param>
 ----
 
-That component uses `keycloak.config` or `karaf.etc` java properties to search for a base folder to locate the configuration.
-Then inside one of those folders it searches for a file called `<your_web_context>-keycloak.json`.
+The available resolvers for `keycloak.config.resolver` are the following ones:
+
+org.keycloak.adapters.osgi.BundleBasedKeycloakConfigResolver::
+  Default resolver that looks up for the configuration file inside the bundle it
+  is configured from. By default, it loads file named `WEB-INF/keycloak.json` but this file name can be configured via `configLocation` property.
+
+org.keycloak.adapters.osgi.PathBasedKeycloakConfigResolver::
+  It searches for a file called `<your_web_context>-keycloak.json` inside a folder
+  that is specified by `keycloak.config` system property. If `keycloak.config` is
+  not set, `karaf.etc` system property is used instead.
++
+For example, if your web application is deployed into context `my-portal`, then
+  your adapter configuration would be loaded either from the 
+  `${keycloak.config}/my-portal-keycloak.json` file, or from `${karaf.etc}/my-portal-keycloak.json`.
+
+org.keycloak.adapters.osgi.HierarchicalPathBasedKeycloakConfigResolver::
+  Similar to `PathBasedKeycloakConfigResolver` above, where for given URI path,
```

Comment:
I suggest starting with a parallel phrase, such as "This resolver is similar to..."